src/main/java/com/github/onsdigital/babbage/configuration/ContentAPI.java

```diff
@@ -19,12 +19,16 @@ public class ContentAPI implements AppConfig {
     private final String contentAPIHost;
     private final String topicsAPIHost;
     private final int maxConnections;
+    private final int pooledConnectionsTimeout;
+    private final int idleConnectionsTimeout;
     private static ContentAPI INSTANCE;
     private static final String DEFAULT_CONTENT_API_HOST = "http://localhost:8082";
     private static final String CONTENT_API_HOST_KEY = "CONTENT_SERVICE_URL";
     private static final String DEFAULT_TOPICS_API_HOST = "http://localhost:25300";
     private static final String TOPICS_API_HOST_KEY = "TOPICS_SERVICE_URL";
     private static final String MAX_CONNECTIONS_KEY = "CONTENT_SERVICE_MAX_CONNECTION";
+    private static final String POOLED_CONNECTION_TIMEOUT_KEY = "POOLED_CONNECTION_TIMEOUT_KEY";
+    private static final String IDLE_CONNECTION_TIMEOUT_KEY = "IDLE_CONNECTION_TIMEOUT_KEY";
```

Comment:
shouldn't these be:
-  "POOLED_CONNECTION_TIMEOUT"
- "IDLE_CONNECTION_TIMEOUT"

Can these new config items be added to the readme as well please?

Reply:
These have now been changed and added to the readme.